python.d/monit.chart.py

```diff
@@ -111,14 +111,14 @@ def _get_data(self):
                 self.debug("Skipping service from 'System' category, because it's useless in graphs")
                 continue
 
-            xpath_query = "./service[@type='%d']" % (service_id)
-            self.debug("Searching for %s as %s" % (service_category, xpath_query))
+            xpath_query = "./service[@type='{:d}']".format(service_id)
+            self.debug("Searching for {} as {}".format(service_category, xpath_query))
```

Comment:
Oh, please  give placeholders an explicit positional index, like  `{0} {1}`.  (py2.6 compatibility)

Reply:
Ahah ... Added. And I hope i understood you right)